package.json

```diff
@@ -78,7 +78,7 @@
     "build-mjs": "rimraf mjs && BABEL_ENV=mjs babel src --ignore __tests__,__mocks__ -d mjs && yarn build-mjs-rename && COPY_TO_FOLDER=mjs npm run build-flow",
     "build-mjs-rename": "find ./mjs -name \"*.js\" -exec bash -c 'mv \"$1\" \"${1%.js}\".mjs' - '{}' \\;",
     "build-node8": "rimraf node8 && BABEL_ENV=node8 babel src --ignore __tests__,__mocks__ -d node8 && COPY_TO_FOLDER=node8 npm run build-flow",
-    "build-flow": "echo `$1` && find ./src -name '*.js' -not -path '*/__*' | while read filepath; do cp $filepath `echo ./${COPY_TO_FOLDER:-lib}$filepath | sed 's/.\\/src\\//\\//g'`.flow; done",
+    "build-flow": "echo `$1` && find ./src -name '*.js' -not -path '*/__*' | while read filepath; do cp $filepath `echo ./${COPY_TO_FOLDER:-lib}$filepath | sed 's/.\\/src\\//\\//g'`.flow; done && find $COPY_TO_FOLDER -name '*.flow' | while read filepath; do sed -i \"s/react-relay-network-modern\\/lib/react-relay-network-modern\\/$COPY_TO_FOLDER/g\" $filepath; done",
```

Comment:
I really don't understand what you want to do here )

Reply:
Command: `find $COPY_TO_FOLDER -name '*.flow' | while read filepath; do sed -i \"s/react-relay-network-modern\\/lib/react-relay-network-modern\\/$COPY_TO_FOLDER/g\" $filepath; done`.

This can replace the `react-relay-network-modern/lib` to `react-relay-network-modern\node8` in `node8`, `react-relay-network-modern/lib` to `react-relay-network-modern/mjs` in `mjs` and so on.

Reply:
Oh, I understood correctly for the first time. 

It does not help!

For node8 and mjs we need to create a new definition file which, which internally import types from `lib/definitidons.js` and re-export them.

How I understood flow throws an Error if different types' definition with the same name and body tracks as different types and from v0.101 starts to throw an error.

It may be fixed in the next flow version, maybe not )

So I think that `any` or `// $FlowFixMe` currently should be our workaround.

PS. At last time I very dislike Flow.

Reply:
> For node8 and mjs we need to create a new definition file which, which internally import types from `lib/definitidons.js` and re-export them.

Why we need to create a new definition file for each type?  As I know, the difference between `node8` and `mjs` is how the module export.

> How I understood flow throws an Error if different types' definition with the same name and body tracks as different types and from v0.101 starts to throw an error.

Can you give the example? As a know, the same name will be seen as the multiple types with this name.